repository/u.json

```diff
@@ -598,10 +598,10 @@
 			{
 			"name": "Urtext",
 			"details": "https://github.com/nbeversl/urtext_sublime",
-			"labels": ["plaintext","notes","writing"],
+			"labels": ["plaintext", "notes", "note-taking", "text", "writing", "zettelkasten"],
 			"releases": [
-				{
-					"sublime_text": ">=3092",
+				{	
+					"sublime_text": ">=3000",
 					"tags": true
```

Comment:
```suggestion
				{
					"sublime_text": ">=3092",
					"tags": true
```

The change in supported versions, going _back_ to support version 3000-3092, doesn't make much sense to me. And the addition of trailing whitespace triggers a bit 😂 Please undo that change.